File: pinyougou-parent/pinyougou-interface/src/main/java/cn/itcast/core/service/ItemCatService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.item.ItemCat;
import entity.PageResult;
import entity.Result;
import vo.ItemCatListVo;

import java.util.List;
import java.util.Map;

public interface ItemCatService {
    List<ItemCat> findByParentId(Long parentId);

    ItemCat findOne(Long id);

    List<ItemCat> findAll();

    //分页查询
    PageResult search(Integer page,Integer rows);

    Result updateStatus(Long[] ids,String status);
}

File: pinyougou-parent/pinyougou-web-portal/src/main/webapp/js/service/itemCatService.js
app.service("itemCatService",function($http){

	this.findItemCatList = function(parentId){
		//alert(parentId);
		return $http.get("itemCat/findItemCatList.do?parentId="+parentId);
	}

    this.findItemCatList2 = function(parentId){
        //alert(parentId);
        return $http.get("itemCat/findItemCatList2.do?parentId="+parentId);
    }
});